README.md

```diff
@@ -29,7 +29,14 @@ make deploy IMG=<some-registry>/terraclassic.operator:tag
 3. Install Instances of Custom Resources:
 
 ```sh
-kubectl apply -f config/samples/
+kubectl apply -f config/test-samples/
```

Comment:
Not sure I like the renaming of this folder. As such when we are done samples should be applicable in any context, if your willing to accept the "sensible defaults" and fill out the "<input-value-here>" placeholders :)